aspnetcore/fundamentals/websockets.md

```diff
@@ -92,6 +92,32 @@ The WebSocket closed exception can also happen when returning too soon from an a
 
 Never use `Task.Wait`, `Task.Result`, or similar blocking calls to wait for the socket to complete, as that can cause serious threading issues. Always use `await`.
 
+::: moniker range=">= aspnetcore-6.0"
+
+### Using Compression
+
+> [!WARNING]
+> Enabling compression over encrypted connections makes the application subject to CRIME/BREACH type attacks.
+> If you are sending sensitive information consider not enabling compression, or make sure you use [WebSocketMessageFlags.DisableCompression](TODO) when calling [WebSocket.SendAsync](TODO).
+> This applies to both sides of the WebSocket, it's worth noting that the WebSockets API in the browser doesn't have configuration for disabling compression per send. 
+
+If compression of messages over WebSockets is desired then the accept code needs to specify that it allows compression as follows:
```

Comment:
The suggestion will add a blank line between the sentence and the code block that follows:

```suggestion
If compression of messages over WebSockets is desired, then the accept code must specify that it allows compression as follows:

```